assets/app/view/game/player.rb

```diff
@@ -252,6 +256,12 @@ def render_minors(minors)
         }
         h('div', outer_props, [h('div', inner_props, minor_logos)])
       end
+
+      def num_shares_of(entity, corporation)
```

Comment:
don't copy and paste code, make this a module

Reply:
I made this a module underneath `app/view` where both of the uses are from